app/src/main/java/de/mwvb/blockpuzzle/LocaleHelper.java

```diff
@@ -0,0 +1,68 @@
+package de.mwvb.blockpuzzle;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.preference.PreferenceManager;
+
+import java.util.Locale;
+
+public class LocaleHelper {
```

Comment:
Someone once said to me: »all classes that end in "er" are crap«, e.g. "Helper", "Manager". Suggestion: LocaleService